docs/ert/getting_started/howto/design_matrix.rst

```diff
@@ -0,0 +1,36 @@
+Design matrix
+=============
+
+The `DESIGN_MATRIX` keyword in ERT allows users to define a design matrix in Excel,
+as described in the :ref:`design_matrix` section of the documentation. The design matrix
+contains parameters and their values per realization.
+
+`DESIGN2PARAMS` is a deprecated forward model and users are encouraged to migrate to the
+`DESIGN_MATRIX` keyword for defining design matrices.
+
+Migration from `DESIGN2PARAMS`
+------------------------------
+
+Migrating from the `DESIGN2PARAMS` forward model involves updating your configuration to use the `DESIGN_MATRIX` keyword.
+The `DESIGN_MATRIX` keyword replaces `DESIGN2PARAMS` by integrating design matrix functionality directly into ERT,
+offering improved validation and usability.
```

Comment:
integrating design matrix parameters directly into ERT
offering improved validation and parameter usability (like plotting) in ert.